packages/block-library/src/social-link/index.php

```diff
@@ -15,7 +15,7 @@
  * @return string Rendered HTML of the referenced block.
  */
 function render_block_core_social_link( $attributes, $content, $block ) {
-	$open_in_new_tab = $block->context['openInNewTab'];
+	$open_in_new_tab = array_key_exists( 'openInNewTab', $block->context ) ? $block->context['openInNewTab'] : false;
```

Comment:
Why not use `isset` here like the rest of code? I think that way you wouldn't have problems even if `$block-context` doesn't exist.

Reply:
It would be great to update and land this PR, I saw this notice recently.

Reply:
@gziolo Thanks for putting the note here. I just updated the PR :)